src/dmd/root/strtold.d

```diff
@@ -0,0 +1,517 @@
+/**
+ * Compiler implementation of the
+ * $(LINK2 http://www.dlang.org, D programming language).
+ *
+ * Copyright:   Copyright (C) 1985-1998 by Symantec
+ *              Copyright (C) 2000-2019 by The D Language Foundation, All Rights Reserved
+ * Authors:     $(LINK2 http://www.digitalmars.com, Walter Bright)
+ * License:     $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Source:      $(LINK2 https://github.com/dlang/dmd/blob/master/src/dmd/backend/strtold.c, backend/strtold.c)
+ */
+
+module dmd.root.strtold;
+
+import dmd.root.longdouble;
+import core.stdc.ctype;
+import core.stdc.errno;
+
+version(CRuntime_Microsoft):
+@nogc:
+nothrow:
+
+static if (false)
+{
+/* This is for compilers that support 80-bit floats,
```

Comment:
DMD supports 80-bit float, can we just use this and drop the fallback?

Reply:
When compiling with LDC on Windows, `real`s are 64-bit floats.

Reply:
OK.